addons/l10n_lu/account.account.template-2011.csv

```diff
@@ -1025,17 +1025,17 @@ lu_2011_account_73314,73314,Constructions sur sol d'autrui,lu_2011_account_7331,
 lu_2011_account_7332,7332,Installations techniques et machines,lu_2011_account_733,other,account_type_2011_7_produit_exploitation,FALSE,
 lu_2011_account_7333,7333,"Autres installations, outillage, mobilier et matériel roulant",lu_2011_account_733,other,account_type_2011_7_produit_exploitation,FALSE,
 lu_2011_account_7334,7334,Acomptes versés et immobilisations corporelles en cours,lu_2011_account_733,other,account_type_2011_7_produit_exploitation,FALSE,
-lu_2011_account_734,734,Reprises de corrections de valeur sur stocks,lu_2011_account_73,view,account_type_2011_7_produit_exploitation,FALSE,"account_financial_report_146"
+lu_2011_account_734,734,Reprises de corrections de valeur sur stocks,lu_2011_account_73,view,account_type_2011_7_produit_exploitation,FALSE,"account_financial_report_146,account_financial_report_abr_146"
 lu_2011_account_7341,7341,Matières premières et consommables,lu_2011_account_734,other,account_type_2011_7_produit_exploitation,FALSE,
 lu_2011_account_7342,7342,Produits en cours de fabrication et commandes en cours,lu_2011_account_734,other,account_type_2011_7_produit_exploitation,FALSE,
 lu_2011_account_7343,7343,Produits finis et marchandises,lu_2011_account_734,other,account_type_2011_7_produit_exploitation,FALSE,
 lu_2011_account_7344,7344,Terrains et immeubles destinés à la revente,lu_2011_account_734,other,account_type_2011_7_produit_exploitation,FALSE,
 lu_2011_account_7345,7345,Acomptes versés,lu_2011_account_734,other,account_type_2011_7_produit_exploitation,FALSE,
-lu_2011_account_735,735,Reprises de corrections de valeur sur créances de l'actif circulant,lu_2011_account_73,view,account_type_2011_7_produit_exploitation,FALSE,"account_financial_report_146"
+lu_2011_account_735,735,Reprises de corrections de valeur sur créances de l'actif circulant,lu_2011_account_73,view,account_type_2011_7_produit_exploitation,FALSE,"account_financial_report_146,account_financial_report_abr_146"
 lu_2011_account_7351,7351,Créances résultant de ventes et prestations de services,lu_2011_account_735,other,account_type_2011_7_produit_exploitation,FALSE,
 lu_2011_account_7352,7352,Créances sur des entreprises liées et sur des entreprises avec lesquelles la société a un lien de participation,lu_2011_account_735,other,account_type_2011_7_produit_exploitation,FALSE,
 lu_2011_account_7353,7353,Autres créances,lu_2011_account_735,other,account_type_2011_7_produit_exploitation,FALSE,
-lu_2011_account_74,74,Autres produits d'exploitation,lu_2011_account_7,view,account_type_2011_7_produit_exploitation,FALSE,"account_financial_report_147,account_financial_report_abr_147"
+lu_2011_account_74,74,Autres produits d'exploitation,lu_2011_account_7,view,account_type_2011_7_produit_exploitation,FALSE,"account_financial_report_147,account_financial_report_abr_141,account_financial_report_abr_120"
```

Comment:
The same account in two sections of the same financial report (abr) looks suspicious to me. @laetitia-gangloff can you double check?